project/Game.py

```diff
@@ -140,7 +140,12 @@ def answer_question(self, team, answer):
         return True
 
     def get_status(self, team):
-        return ""
+        currenttimecalc = (team.clueTime + datetime.timedelta(hours=datetime.datetime.now().hour,minutes=datetime.datetime.now().minute,seconds=datetime.datetime.now().second))
```

Comment:
You are calculating datetime.now() multiple times. It would be better to just do it once, store it in a variable and use that where needed.